tests/unit/OptionsListUtilsTest.js

```diff
@@ -567,39 +572,57 @@ describe('OptionsListUtils', () => {
     });
 
     it('getShareDestinationsOptions()', () => {
+        // Filter current REPORTS as we do in the component, before getting share destination options
+        const filteredReports = {};
+        _.keys(REPORTS).forEach((reportKey) => {
+            if (ReportUtils.shouldDisableWriteActions(REPORTS[reportKey]) || ReportUtils.isExpensifyOnlyParticipantInReport(REPORTS[reportKey])) {
+                return;
+            }
+            filteredReports[reportKey] = REPORTS[reportKey];
```

Comment:
we have to filter here b/c we do this in `TaskShareDestinationSelectorModal` before calling `getShareDestinationOptions`. So we either do this logic in the test as well, OR we extract this filtering into `getShareDestinationOptions`

Reply:
I am not able to understand why we have to do this change? Because `getShareDestinationOptions` is a separate function not dependent on any component right?

Reply:
I believe we need the change b/c we currently use `getShareDestinationOptions` by passing in filtered reports - here:

https://github.com/Expensify/App/blob/4a7558cdfea721e58675b2e1d1f9ba25a2fd9e31/src/pages/tasks/TaskShareDestinationSelectorModal.js#L49-L60

So tests were failing when we weren't filtering out archived reports (via `ReportUtils.shouldDisableWriteActions) first